amadeus-streaming/Cargo.toml

```diff
@@ -27,6 +27,10 @@ rand = { version = "0.7", features = ["small_rng"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_closure = "0.3"
 twox-hash = "1.1"
+protobuf = "2.22.1"
```

Comment:
Would it be possible to make `protobuf` an optional dependency? And re-expose the option at the top crate also.

Reply:
Added the feature "ser-proto"